apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -6099,16 +6096,8 @@ highcharts-chart {
     top: 0 !important;
 }
 .voucher-preview-edit {
-    .primary-main-header {
```

Comment:
This is reverting PR https://github.com/Walkover-Web-Solution/Giddh-New-Angular4-App/pull/9980 changes